packages/nvidia_nat_test/src/nat/test/plugin.py

```
@@ -722,3 +722,52 @@ def oauth2_client_credentials_fixture(oauth2_server_url: str, fail_missing: bool
+def piston_url_fixture(fail_missing: bool) -> str:
+    """
+    Configuration for piston testing.
+
+    The public piston server limits usage to five requests per minute.
+    """
+    import requests
+    url = os.environ.get("NAT_CI_PISTON_URL", "https://emkc.org/api/v2/piston")
+    try:
+        response = requests.get(f"{url.rstrip('/')}/runtimes", timeout=30)
+        response.raise_for_status()
+        return url
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)